README.md

```diff
@@ -1,2 +1,31 @@
-# advisory-group
-Enterprise Advisory Group
+# Enterprise Advisory Group
+
+The Enterprise Advisory Group is a group of corporate representatives in roles of leadership responsible managing for large-scale deployments of Node.js in an Enterprise environment. As a group, we strive to provide timely feedback to the [TSC](https://github.com/nodejs/TSC), the [CommComm](https://github.com/nodejs/community-committee), the [Node.js project](https://github.com/nodejs/node) and the Node.js Foundation about the unique needs surrounding running Node.js in an Enterprise environment.
```

Comment:
i think listing the TSC, the CC and the entire nodejs project is kind of redundant maybe? i believe `the Node.js project` could be left out

Reply:
`Group is a group of` -> `Groups consists of`
`roles of leadership` -> `leadership roles`
`responsible managing` -> `responsible for managing`
`As a group, we strive` -> `We strive`
`unique needs surrounding running Node.js in an Enterprise environment` -> `needs of Node.js Enterprise users`

Reply:
@pup This group is largely comprised of individuals who do not work with GitHub in their day-to-day activities. I have attempted to be as descriptive and comprehensive as possible for the benefit of those individuals to set expectations.

Reply:
Thanks, @Trott. Updated with your suggestions.